extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -102,7 +102,7 @@ export class MethodsUsageDataProvider
   }
 
   private getModelingStatusIcon(method: Method): ThemeIcon {
-    const modeledMethods = this.modeledMethods[method.signature];
+    const modeledMethods = this.modeledMethods[method.signature] || [];
```

Comment:
I believe we're using nullish coalescing in most other places:

```suggestion
    const modeledMethods = this.modeledMethods[method.signature] ?? [];
```